experimenter/experimenter/urls.py

```diff
@@ -23,6 +23,7 @@
     re_path(r"^api/v5/", include("experimenter.experiments.api.v5.urls")),
     re_path(r"^api/v6/", include("experimenter.experiments.api.v6.urls")),
     re_path(r"^admin/", admin.site.urls),
+    re_path(r"^history/", include("experimenter.changelog.urls")),
```

Comment:
keep it consistent, somewhere we are saying history, somewhere we are saying changelogs, as we defined in URLs, would suggest changing history to changelog

Reply:
you can also keep this :)